CT-1782 add tests for deleting an ICO case

beccapearce opened the pull request:
This tests deleting an open ICO we should add a test for closed
ICOs once they're available following the same pattern as the test for
deleting a SAR case

aliuk2012 commented on app/views/messages/_speech_bubble.html.slim:
```diff
@@ -5,5 +5,6 @@
-      = message_event.created_at.strftime("%d/%m/%Y %H:%M")
+      = " - "
+      = message_event.messaging_team&.name || "Team unknown"
+      p = message_event.created_at.strftime("%d/%m/%Y %H:%M")
```
```
br
= message_event.created_at.strftime("%d/%m/%Y %H:%M")
```

Commit:
CT-1782 add tests for deleting an ICO case

This tests deleting an open ICO we should add a test for closed
ICOs once they're available following the same pattern as the test for
deleting a SAR case

aliuk2012 reviewed (commented):
Just for consistency, could `spec/features/cases/ico/deleting_ico_spec.rb` be renames to case_deleting_spec.rb like SARS or perhaps updates SAR.

Commit:
CT-1782 add delete tests for SARs
